genie/clinical.py

```diff
@@ -322,12 +322,18 @@ def validate_helper(self, clinicalDF, clinicalSampleDF, oncotreeLink):
 			seqAssayIds = clinicalSampleDF.SEQ_ASSAY_ID[clinicalSampleDF.SEQ_ASSAY_ID != ""]
 			allSeqAssays = seqAssayIds.unique()
 			notNormalized = []
+			not_caps = []
 			for seqassay in allSeqAssays:
-				checkIfNormalized = [bool(re.search(seqassay,seq,re.IGNORECASE)) for seq in allSeqAssays]
+				#Add ^ and $ to make sure that strings match exactly, allow for V1 and V1.1...
+				checkIfNormalized = [bool(re.search("^" + seqassay + "$",seq,re.IGNORECASE)) for seq in allSeqAssays]
```

Comment:
Remove this, because all SEQ_ASSAY_IDs are capitalized. 
Remove adding of CENTER to SEQ_ASSAY_ID from above